### ISSUE TITLE
A comment about dependencies...

### ISSUE THREAD

**Issue by bredele** [tether/roach#3]
I'm being careful about two things:
- dependency is released and tested
- dependency can be used in client side*

\* Even if roach is a server side project, you could potentially use some part of it in a browser (like the crawler part).

**Comment by ekryski** [comment]
Yes I think that is really cool however, it isn't a priority in the foreseeable future for these to be run client side so I wouldn't spend time on it specifically.

**Comment by ekryski** [comment]
However the dependency being released and tested is totally awesome and should be the case. For now though we can keep all the dependencies local to the project. 

We will be open sourcing this version of roach eventually (and some of its components) but for now it should stay all internal and probably under the same project until we have tested it in production.
